docs/tables/aws_cloudwatchlogs_log_groups.md

```diff
@@ -0,0 +1,16 @@
+
+# Table: aws_cloudwatchlogs_log_groups
+Represents a log group.
+## Columns
+| Name        | Type           | Description  |
+| ------------- | ------------- | -----  |
+|account_id|text|The AWS Account ID of the resource.|
+|region|text|The AWS Region of the resource.|
+|tags|jsonb||
```

Comment:
description required (should be added to the `gen.hcl` file)

Reply:
Added!